### PR TITLE
service: Add a README.md to nydus-service

### PR DESCRIPTION
**Pull request by taoohong**
Add a README.md to nydus-service crate.

### CODE REVIEW

File: service/README.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+```
+
+**create a daemon**:
+
+```Rust
+static ref DAEMON_CONTROLLER: DaemonController = DaemonController::default()
+
+let cmd = FsBackendMountCmd {
+    fs_type: FsBackendType::Rafs,
+    //Bootstrap path
```
Comment (imeoer) [inline service/README.md:73]: Should add the space after `//`.

File: service/README.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# nydus-service
```
Comment (imeoer) [inline service/README.md:1]: Could we put a link to reference it in [README.md](https://github.com/dragonflyoss/image-service/blob/master/README.md)?

File: service/README.md (hunk 1 above)
Comment (taoohong) [reply service/README.md:73]: Thanks for your reply, I'll fix it.

File: service/README.md (hunk 2 above)
Comment (taoohong) [reply service/README.md:1]: What about adding a part named "Reuse Nydus Services" in [README.md](https://github.com/dragonflyoss/image-service/blob/master/README.md) to put this link.

Comment (imeoer) [reply service/README.md:1]: Good, and give a brief description.

**Comment by jiangliu** [comment]
Let's delay this PR to v2.3?

**Comment by taoohong** [comment]
> Let's delay this PR to v2.3?

Ok. that's fine.

File: service/README.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+  },
+  "mode": "direct",
+  "digest_validate": false,
+  "iostats_files": false,
+  "enable_xattr": true,
+  "fs_prefetch": {
+    "enable": true,
+    "threads_count": 4
+  }
+}
```
Comment (jiangliu) [inline service/README.md:63]: Could you please help to update config_v2 format?

Comment (jiangliu) [reply service/README.md:63]: https://github.com/dragonflyoss/image-service/blob/master/docs/samples/configuration_v2.toml

Comment (taoohong) [reply service/README.md:63]: Yeah, I'd like to.

Comment (adamqqqplay) [reply service/README.md:63]: @taoohong Hi, have you solved this problem?

**Review by ccx1024cc — COMMENTED** [review]
Please also backport to stable/v2.2. Thanks ~~~~

**Comment by taoohong** [comment]
> Please also backport to stable/v2.2. Thanks ~~~~

Ok

**Review by jiangliu — APPROVED** [review]

**Pull request merged**